lib/signatures.js

```diff
@@ -50,9 +50,16 @@ function getSignatures( key ) {
 }
 
 function getSignatureErrors() {
-	return all().then( function( signatures ) {
-		return validate( signatures ).errors;
-	} );
+	return Promise.all( [
+		module.exports.raw( keys.cla ),
+		module.exports.raw( keys.caa )
+	] )
+		.then( function( signatures ) {
+			return signatures[ 0 ].concat( signatures[ 1 ] );
+		} )
+		.then( function( signatures ) {
+			return validate( signatures );
+		} );
```

Comment:
That's correct, I'm working on a replacement. Wanted to start by increasing test coverage on this method. I'm now rewriting the method return a different data structure. Once all uses are switched over, I'll remove the current `hashed` method.

I guess at this point the added tests are more interesting than the implementation changes...